vocabularies/Analytics.json

```diff
@@ -161,6 +164,114 @@
         "@Validation.ApplicableTerms@Core.Description": "Adding a list of other terms that can be annotated to it.",
         "@Validation.ApplicableTerms": ["Common.Label"]
       }
+    },
+    "AutoExpand": [
+      {
+        "$Kind": "Function",
+        "$EntitySetPath": "InputSet",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.IsSignature": true,
+        "@Core.Description": "`$apply` transformation that expands an unnamed leveled hierarchy with custom aggregation of certain properties",
+        "$Parameter": [
+          {
+            "$Name": "InputSet",
+            "$Collection": true,
+            "$Type": "Edm.EntityType",
+            "@Core.Description": "Entity set to be processed"
+          },
+          {
+            "$Name": "Levels",
+            "$Collection": true,
+            "$Type": "Analytics.AutoExpandLevel",
+            "@Core.Description": "Collection of aggregation levels forming a leveled hierarchy",
+            "@Core.LongDescription": "Each element in the collection defines the property names for one level.\n            A property must not referenced by more than one level.\n            The first element in the collection defines the property names of the coarsest level,\n            the following elements define the property names of consecutively finer-grained aggregation levels.\n            All referenced properties must be groupable."
+          },
+          {
+            "$Name": "ExpandLevel",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number of levels to be expanded, counting from left",
+            "@Core.LongDescription": "The function result comprises the leveled hierarchy with the expanded levels and the following level as leaves in preorder."
+          },
+          {
+            "$Name": "Aggregation",
```

Comment:
[sap.ui.model.odata.v4.ODataListBinding#setAggregation](https://ui5.sap.com/#/api/sap.ui.model.odata.v4.ODataListBinding%23methods/setAggregation) foresees to specify the `unit` with the related unit or currency property for a property in `aggregate`. Is this required here too or would the backend know that related unit/currency property and to the right thing automatically (depending on whether the unit/currency is used for grouping or not)?

Reply:
If the unit is a custom aggregate (which produces `null` if there is no unique value), then it must be passed in the `Aggregation` parameter. If it is used for grouping, it must be passed in the `Levels` parameter. How could the backend know which alternative the client desires?